src/Fieldtypes/Terms.php

```diff
@@ -209,6 +209,8 @@ protected function toItemArray($id)
             return $this->invalidItemArray($id);
         }
 
+        $term = $term->in(Site::current()->handle());
```

Comment:
There doesn't seem to be any way to get the site from the request, but the current site (or the main one) is better than just the last one

Reply:
Everything seems to work fine without the addition of this line. Is it required?

Reply:
Yes, without this, I was getting the FR (second site) taxonomy translations displayed in the "already selected" taxonomies

![image](https://user-images.githubusercontent.com/3626559/107742518-76342d00-6d0f-11eb-9f73-bb3a475eec84.png)

With this it's the NL ones (first and default site).

Ideally it would show the NL translations on the NL version of the entry, and FR versions on the FR entry, but I couldn't find right away how to get that, so I opted that the current/default site was better.

Reply:
Thanks for confirming. I pushed a commit that'll get the locale from the entry, if you wanted to see how that's done.